composer.json

```diff
@@ -163,6 +163,12 @@
             "Composer\\Config::disableProcessTimeout",
             "php matchbot-cli.php messenger:consume -vv --time-limit=7080"
         ],
+        "save-build-commit-id": [
+            "echo '<?php \nreturn \"'`git rev-parse HEAD`'\";\n' > .build-commit-id.php"
+        ],
+        "post-install-cmd": [
+            "composer save-build-commit-id"
+        ],
```

Comment:
tested by sshin'g into circleCI's build container:

```
root@6b92a6681b73:/var/www/html# cat .build-commit-id.php   
<?php 
return "0531dd6fb7981a4fd92220e4955f4fe534f81def";

```

So a file like that should now be present in every docker image we build.